pgmpy/estimators/expert.py

```diff
@@ -250,6 +270,15 @@ def estimate(
                 edge_direction = (selected_edge.u, selected_edge.v)
             elif (selected_edge.v, selected_edge.u) in orientations:
```

Comment:
I would suggest mentioning in the docstring that `orientations` has priority over `temporal_order`.